system/Helpers/form_helper.php

```diff
@@ -0,0 +1,954 @@
+
+if ( ! function_exists('set_radio'))
+{
+    /**
+     * Set Radio
+     *
+     * Let's you set the selected value of a radio field via info in the POST array.
+     * If Form Validation is active it retrieves the info from the validation class
+     *
+     * @param	string	$field
+     * @param	string	$value
+     * @param	bool	$default
+     * @return	string
+     */
+    function set_radio(string $field, string $value = '', bool $default = FALSE): string
+    {
+        // Form inputs are always strings ...
+        $value = (string) $value;
+        
+        $input = $_POST[$field];
+        
+        if (is_array($input))
+        {
+            // Note: in_array('', array(0)) returns TRUE, do not use it
+            foreach ($input as &$v)
+            {
+                if ($value === $v)
+                {
+                    return ' checked="checked"';
+                }
+            }
+            return '';
+        }
+        
+        // Unchecked checkbox and radio inputs are not even submitted by browsers ...
+        if ($_POST)
+        {
+            return ($input === $value) ? ' checked="checked"' : '';
+        }
+        
+        return ($default === TRUE) ? ' checked="checked"' : '';
+    }
+}
+
+// ------------------------------------------------------------------------
+
+if ( ! function_exists('_parse_form_attributes'))
+{
+    /**
+     * Parse the form attributes
+     *
+     * Helper function used by some of the form helpers
+     *
+     * @param	array	$attributes	List of attributes
+     * @param	array	$default	Default values
+     * @return	string
+     */
+    function _parse_form_attributes($attributes, $default): string
+    {
+        if (is_array($attributes))
+        {
+            foreach ($default as $key => $val)
+            {
+                if (isset($attributes[$key]))
+                {
+                    $default[$key] = $attributes[$key];
+                    unset($attributes[$key]);
+                }
+            }
+            if (count($attributes) > 0)
+            {
+                $default = array_merge($default, $attributes);
+            }
+        }
+        
+        $att = '';
+        
+        foreach ($default as $key => $val)
+        {
+            if ($key === 'value')
+            {
+                $val = htmlspecialchars($val);
+            }
+            elseif ($key === 'name' && ! strlen($default['name']))
+            {
+                continue;
+            }
+            $att .= $key.'="'.$val.'" ';
+        }
+        
+        return $att;
+    }
+}
+
+// ------------------------------------------------------------------------
+
+if ( ! function_exists('_attributes_to_string'))
+{
+    /**
+     * Attributes To String
+     *
+     * Helper function used by some of the form helpers
+     *
+     * @param	mixed
+     * @return	string
+     */
+    function _attributes_to_string($attributes): string
```

Comment:
I believe we can remove this method and use the [stringify_attributes](https://github.com/bcit-ci/CodeIgniter4/blob/develop/system/Common.php#L625) that is in the Common file.... 

Please double-check, but at a glance, they appear the same.